images/win/scripts/Installers/Download-ToolCache.ps1

```diff
@@ -1,51 +1,62 @@
 ################################################################################
 ##  File:  Download-ToolCache.ps1
+##  Team:  CI-Build
 ##  Desc:  Download tool cache
 ################################################################################
 
-Function InstallTool
-{
-    Param
-    (
-        [System.Object]$ExecutablePath
+Function Install-NpmPackage {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)]
+        [System.String]
+        $Name,
+        [Parameter(Mandatory=$true)]
+        [System.String]
+        $NpmRegistry
     )
 
-    Write-Host $ExecutablePath.DirectoryName
-    Set-Location -Path $ExecutablePath.DirectoryName
-    Get-Location | Write-Host
-    if (Test-Path 'tool.zip')
-    {
+    Write-Host "Installing npm '$Name' package from '$NpmRegistry'"
+
+    npm install $Name --registry=$NpmRegistry
+}
+
+Function InstallTool {
```

Comment:
Do we need this function? there will be no `tool.zip` file to unzip, also, I couldn't find from where this function can be called.

Reply:
Yep, we don't need it. It is a leftover from previous installation method, I'll remove it. Thanks for the observation.